src/unpoly/classes/event_listener.js

```diff
@@ -90,6 +90,10 @@ up.EventListener = class EventListener extends up.Record {
         args.push(data)
       }
 
+      if (element instanceof HTMLButtonElement && element.disabled) {
+        return
+      }
```

Comment:
I see two issues here:

1. This ignores all types of events on disabled buttons, but we must only ignore `click`. We should add a check on `this.eventType` here.
2. This only ignores events on `<button>` elements, but we must also ignore them for `<input type="button">` or any other kind of form control. Better remove the `instanceof` check and just test if `element.disabled`. This is also what [jQuery does](https://github.com/jquery/jquery/blob/27303c6be09b8fc24c13454deae234e480cbf995/src/event.js#L363-L364).

Reply:
Thanks for the review. I agree with this and will make the changes.